WalletWasabi/Hwi/Models/HwiCommands.cs

```diff
@@ -4,20 +4,125 @@
 
 namespace WalletWasabi.Hwi.Models
 {
+	/// <summary>
+	/// List of commands that HWI project supports for various <see href="https://en.wikipedia.org/wiki/Human_interface_device">HID devices</see>.
+	/// </summary>
+	/// <remarks>HWI may partially support a hardware device.</remarks>
+	/// <seealso href="https://github.com/bitcoin-core/HWI/blob/master/hwilib/hwwclient.py"/>
+	/// <seealso href="https://github.com/bitcoin-core/HWI/blob/master/docs/trezor.md"/>
+	/// <seealso href="https://github.com/bitcoin-core/HWI/blob/master/docs/ledger.md"/>
 	public enum HwiCommands
 	{
+		/// <summary>Get a list of all available hardware wallets.</summary>
 		Enumerate,
+
+		/// <summary>
+		/// Return the master BIP44 public key.
+		///
+		/// <para>Retrieve the public key at the "m/44h/0h/0h" derivation path.</para>
+		/// </summary>
+		/// <remarks>
+		/// Return <code>{"xpub": &lt;xpub string&gt;"}</code>.
+		/// </remarks>
 		GetMasterXpub,
+
+		/// <summary>
+		/// Sign a partially signed bitcoin transaction (PSBT).
+		/// </summary>
+		/// <remarks>
+		/// Return <code>{"psbt": &lt;base64 psbt string&gt;</code>.
+		/// </remarks>
 		SignTx,
+
+		/// <summary>
+		/// ???
+		/// </summary>
 		GetXpub,
+
+		/// <summary>
+		/// Sign a message (bitcoin message signing).
+		///
+		/// <para>Sign the message according to the bitcoin message signing standard.</para>
+		/// <para>Retrieve the signing key at the specified BIP32 derivation path.</para>
+		/// </summary>
+		/// <remarks>
+		/// Return <code>{"signature": &lt;base64 signature string&gt;}</code>.
+		/// </remarks>
 		SignMessage,
+
+		/// <summary>
+		/// ??? (unused at the moment?)
+		/// </summary>
```

Comment:
Returns a list of output descriptors for the specified account, index and count. This is not useful for us because we have no key pools but a HD wallet instead.